crates/kernel/src/tasks/command_parse.rs

```diff
@@ -45,14 +45,14 @@ pub struct ParsedCommand {
     pub argstr: String,
     pub args: Vec<Var>,
 
-    pub dobjstr: String,
-    pub dobj: Objid,
+    pub dobjstr: Option<String>,
```

Comment:
When these aren't set, is it equivalent to setting the perms to `None None None`? I thought the MOO semantics was  to defailt to `this none this to avoid `it being callable as a verb

Reply:
These are for the command invocation parsing, not the verb definition. I had it before with defaulting to empty strings and Objid(-1) ($nothing) which isn't very Rusty? Basically just a way of saying "user typed a command and there was no direct object"